vm/src/builtins/bool.rs

```diff
@@ -110,12 +111,11 @@ impl Constructor for PyBool {
 #[pyclass(with(Constructor, AsNumber, Representable))]
 impl PyBool {
     #[pymethod(magic)]
-    fn format(obj: PyObjectRef, format_spec: PyStrRef, vm: &VirtualMachine) -> PyResult<PyStrRef> {
-        if format_spec.is_empty() {
-            obj.str(vm)
-        } else {
-            Err(vm.new_type_error("unsupported format string passed to bool.__format__".to_owned()))
-        }
+    fn format(obj: PyObjectRef, spec: PyStrRef, vm: &VirtualMachine) -> PyResult<String> {
+        let new_bool = obj.try_to_bool(vm).unwrap();
```

Comment:
Does this work?
```suggestion
        let new_bool = obj.try_to_bool(vm)?;
```

Reply:
Yes, still not totally used to all the Rust shorthands :D